repos/system_upgrade/common/libraries/persistentnetnames.py

```diff
@@ -42,9 +42,13 @@ def interfaces():
             attrs['name'] = dev.sys_name
             attrs['devpath'] = dev.device_path
             attrs['driver'] = dev['ID_NET_DRIVER']
```

Comment:
note it's has been discovered another situation, when ID_NET_DRIVER is not defined and it'S not related to PCI/non-PCI issue.